index.html

```diff
@@ -29,22 +30,208 @@ <h1>
+</div>
+
+  <!--grid section-->
+  <section class="works">
+      
+      <div class="heading">
+        <h1>
+          My Recent <br>Works
+        </h1>
+      </div>
+        <div class="indicator"></div>
+      
+<!--1-->
+        <div class="container">
+          <div class="item">
+            <div class="border">
+              <div class="border-image">
+              <img src="images/grid.svg" alt="">
+            </div>
+            </div>
+            <div class="content">
+              <div class="title"><p>
+            Multi-Post Stories <br>Gain+Glory
+            </p></div>
+            <ul class="links">
+              <li><a href="#">Ruby on Rails</a></li>
+              <li><a href="#">CSS</a></li>
+              <li><a href="#">JavaScript</a></li>
+              <li><a href="#">HTML</a></li>
+          </ul>
+          <div class="button"><a href="#">See project</a></div>
+        </div>
+        </div>
+<!--2-->
+        <div class="item">
+          <div class="border">
+            <div class="border-image">
+            <img src="images/grid.svg" alt="">
+          </div>
+          </div>
+          <div class="content">
+            <div class="title"><p>
+          Multi-Post Stories <br>Gain+Glory
+          </p></div>
+          <ul class="links">
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">JavaScript</a></li>
+            <li><a href="#">HTML</a></li>
+        </ul>
+        <div class="button"><a href="#">See project</a></div>
+      </div>
+      </div>
+<!--3-->
+      <div class="item">
+        <div class="border">
+          <div class="border-image">
+          <img src="images/grid.svg" alt="">
+        </div>
+        </div>
+        <div class="content">
+          <p>
+            Multi-Post Stories <br>Gain+Glory
+            </p> 
+        <ul class="links">
+          <li><a href="#">Ruby on Rails</a></li>
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">JavaScript</a></li>
+          <li><a href="#">HTML</a></li>
+      </ul>   
+      <div class="button"><a href="#">See project</a></div>
+    </div>
+
+<!--4-->
+    <div class="item">
+      <div class="border">
+        <div class="border-image">
+        <img src="images/grid.svg" alt="">
+      </div>
+      </div>
+      <div class="content">
+        <p>
+          Multi-Post Stories <br>Gain+Glory
+          </p> 
+      <ul class="links">
+        <li><a href="#">Ruby on Rails</a></li>
+        <li><a href="#">CSS</a></li>
+        <li><a href="#">JavaScript</a></li>
+        <li><a href="#">HTML</a></li>
+    </ul>
+    <div class="button"><a href="#">See project</a></div>
+  </div>
+
+<!--5-->
+  <div class="item">
+    <div class="border">
+      <div class="border-image">
+      <img src="images/grid.svg" alt="">
+    </div>
+    </div>
+    <div class="content">
+      <p>
+        Multi-Post Stories <br>Gain+Glory
+        </p> 
```

Comment:
- [ ] As requested by the previous reviewer, part of the requirement for this milestone is using proper HTML tags for better semantics, According to the requirement, all project titles should be in `hx` tag, So why don't you change all the project title using `p` tag to `h2 or `h3` tag. :blush: